viam_cartographer_internal_test.go

```diff
@@ -495,59 +495,29 @@ func TestCheckIfIMUAndOdometerSupported(t *testing.T) {
 	ctx := context.Background()
 	t.Run("neither IMU nor Odometer supported", func(t *testing.T) {
 		lidar, movementSensor := s.NoLidar, s.MovementSensorNotIMUNotOdometer
-		imuSupported, odometerSupported, err := checkIfIMUAndOdometerSupported(ctx,
+		err := checkIfIMUAndOdometerSupported(ctx,
 			s.SetupDeps(lidar, movementSensor), string(movementSensor))
-		test.That(t, err, test.ShouldBeNil)
-		test.That(t, imuSupported, test.ShouldBeFalse)
-		test.That(t, odometerSupported, test.ShouldBeFalse)
-	})
-
-	t.Run("only IMU supported", func(t *testing.T) {
```

Comment:
Why were these tests removed?

Reply:
Good point. I could still check for these cases!